src/js/components/sharedComponents/TooltipWrapper.jsx

```diff
@@ -91,41 +94,54 @@ export default class TooltipWrapper extends React.Component {
     }
 
     measureOffset() {
+        let tooltipWidth = 375;
         const tooltipContainer = this.tooltipContainer;
+        const ttContainerWidth = tooltipContainer.clientWidth;
+
         const offsetTop = tooltipContainer.offsetTop + this.props.offsetAdjustments.top;
-        let tooltipWidth = 375;
-        // is the tooltip in a section that takes up the full width of the screen?
-        const isTooltipJustifiedRight = (window.innerWidth - tooltipContainer.offsetLeft) < window.innerWidth / 6;
+        const totalSpace = window.innerWidth;
+        const spaceToRight = (totalSpace - tooltipContainer.offsetLeft) - ttContainerWidth;
+        const spaceToLeft = tooltipContainer.offsetLeft;
 
-        if (this.props.wide && isTooltipJustifiedRight) {
-            tooltipWidth = 700;
+        if (this.props.wide && this.props.left) {
+            tooltipWidth = (spaceToLeft > 800)
+                ? 700
+                : spaceToLeft - 100;
         }
         else if (this.props.wide) {
-            // is there at least 801px of space to the left/right for the tooltip?
-            tooltipWidth = (window.innerWidth - (tooltipContainer.offsetLeft + tooltipContainer.clientWidth) > 800)
-                ? 650
-                : window.innerWidth - (tooltipContainer.offsetLeft + tooltipContainer.clientWidth) - 100;
+            tooltipWidth = (spaceToRight > 800)
+                ? 700
+                : spaceToRight - 100;
         }
 
-        let offsetRight = window.innerWidth - tooltipContainer.offsetLeft - tooltipContainer.clientWidth - tooltipWidth - this.props.offsetAdjustments.right;
         if (this.props.left) {
-            offsetRight = (window.innerWidth - tooltipContainer.offsetLeft) + tooltipContainer.clientWidth;
+            const startingPositionLeft = spaceToLeft - tooltipWidth; // minus tooltipWidth b/c right corner of toolTip is flush w/ left edge of toolTip container
+            this.setState({
+                offsetTop,
+                offsetLeft: startingPositionLeft - horizontalPadding,
+                width: tooltipWidth
+            });
+        }
+        else {
+            const startingPositionLeft = spaceToLeft + ttContainerWidth; // plus ttContainerWidth b/c left corner of toolTip is flush w/ right edge of toolTip container
```

Comment:
can we move this comment above this variable. Adding lines greater than the length for ESLint just adds more warnings to scroll through when finding errors in travis.